examples/DrawingInTheAir/DrawingInTheAir.ino

```diff
@@ -166,10 +169,10 @@ void readVectorFromIMU(byte vector[])
     unsigned int i = 0;
 
     /* Wait until button is pressed */
-    while (digitalRead(buttonPin) == HIGH);
+    while (digitalRead(buttonPin) == LOW);
```

Comment:
Notice the link to the button tutorial in the comment at the top; that is how I am assuming they will connect the button. If they connect it a different way, then they didn't follow the instructions :)